pkg/op/server_http_routes_test.go

```diff
@@ -145,7 +145,7 @@ func TestServerRoutes(t *testing.T) {
 				"assertion":  jwtProfileToken,
 			},
 			wantCode: http.StatusOK,
-			contains: []string{`{"access_token":`, `"token_type":"Bearer","expires_in":299}`},
+			contains: []string{`{"access_token":`, `"token_type":"Bearer","expires_in":299,"scope":"openid"}`},
```

Comment:
I'd like a second opinion on this. It's hard for me to trace back and figure out why this test passes without the `offline_access` scope in the response.

Reply:
Hmm but it's weird it does work for client credentials.

Reply:
Any clue at why this is happening?

Reply:
The example OP storage has the JWT scope validation hard-coded to allow only `openid` as a scope and others are stripped.:

https://github.com/zitadel/oidc/blob/87ab01115708a2a05c20cfd26db82cc7e0e8e338/example/server/storage/storage.go#L566-L576

The Client Credentials does not have a similar step.

For me it is fine to keep it like it is now. So if you're ready you can mark the PR for review.